dockerfiles/Oreo.dockerfile

```diff
@@ -1,33 +1,11 @@
-FROM rust:stretch as build
+FROM debian:bullseye-slim
 
-WORKDIR /src
-
-RUN apt update && apt install -y ca-certificates pkg-config libssl-dev
-
-RUN rustup update 1.59.0 && rustup default 1.59.0
-
-COPY ./vendor/orchestra-types-rs /src/components/orchestra-types-rs
-
-COPY ./vendor/orchestra-event-observer /src/components/orchestra-event-observer
-
-COPY ./vendor/stacks-rpc-client /src/components/stacks-rpc-client
-
-COPY ./components/clarity-repl /src/components/clarity-repl
-
-WORKDIR /src/components/orchestra-event-observer
-
-RUN mkdir /out
-
-RUN cargo build --release
-
-RUN cp target/release/orchestra-event-observer /out
-
-FROM debian:stretch-slim
+COPY orchestra-event-observer /bin/orchestra-event-observer
```

Comment:
Should we be keeping a dockerfile that would let developers building their own docker image? With the current changes, the only way to build a docker image would be via GitHub, right?
I was thinking about moving all the dockerfiles at play in this repo in the `dockerfiles` folder (including `./Dockerfile`, that would be renamed to `./dockerfiles/Clarinet.dockerfile`.
So we could add a `dockerfiles/ci/Oreo.dockerfile` or something?

Reply:
> With the current changes, the only way to build a docker image would be via GitHub, right? 

No, it would still be possible to build one locally, but it would not be as straight forward. It would take a few commands now instead of one.

Having two separate Dockerfiles would certainly maintain ease of access to create one locally, but it's good to be aware that this would mean more overhead for anyone updating the build process. Any changes made to CI would likely have to propagate down to these developer images as well to avoid diverging build processes.

Despite this, I still think it's well worth it to maintain both sets of Dockerfiles.